README.md

```diff
@@ -15,6 +16,38 @@
 This project part of a Service-Oriented Architecture (SOA) application. The Weather-API serves as an intermediary, or microservice, from the Open Weather API and the GTFO-BE repo.
 
 ## How to Install Weather-API
+## API Contract
+
+The weather API adheres to the standard JSON structure outlined on [jsonapi](#jsonapi.org).
+
+- An example of a successful response is as follows:
+```
+{:data=>
+  [
+    {
+      :type=>"weather",
+      :id=>nil,Â
+      :attributes=>
+      {
+       :min_f=>30.400000000000034, 
+       :max_f=>34.99000000000001, 
+       :min_c=>-1.599999999999966, 
+       :max_c=>2.990000000000009, 
+       :day_feels_like_c=>-7.0499999999999545, 
+       :day_feels_like_f=>24.950000000000045, 
+       :description=>"clear sky", 
+       :date=>"2021-01-09T02:00:00+00:00"
+      }
+    }
+  ]
+}
+```
+- An example of an unsuccessful response is:
+```
+{:cod=>"400", :message=>"wrong longitude"}
+```
+
+
 ## Dependencies
 ## Testing
```

Comment:
Second what Jesse and Sheryl mentioned, this looks great.

Reply:
I like that this is included, as well! Thanks